internal/npm_package/test/bundle/BUILD.bazel

```diff
@@ -0,0 +1 @@
+exports_files(["bundle.min.js"])
```

Comment:
this should be a genrule that copies the input file to an output location - we want the packager.js to copy `deps` from the bazel-bin or bazel-genfiles folders.
As you have it here, there's no reason bundle.min.js couldn't be in `srcs`

Reply:
Added genrule to generate `bundle.min.js` and added check in `packager.js` to make sure files in `srcs` reside in `baseDir`.